config/prometheus/collector_alerts.yaml

```diff
@@ -28,6 +28,27 @@ spec:
       labels:
         service: collector
         severity: Warning
+    - alert: CollectorHigh403ForbiddenResponseRate
```

Comment:
@xperimental @cahartma @Clee2691 We should consider the name here.  At minimum it should probably be prefixed as "LogCollector" and I'm wondering what we can do to remove "high" from the name.  Is there a clever way to include the actual threshold value in it? Could we maybe remove it all-together

Reply:
I don't see the problem with this name for the alert. This alert is specific to 403 errors and that there is a high rate of them. 

Even in the [prometheus alerting rules](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) page, the example on the page has the name of `HighRequestLatency`.